accounts-bench/src/main.rs

```diff
@@ -69,11 +70,14 @@ fn main() {
     if fs::remove_dir_all(path.clone()).is_err() {
         println!("Warning: Couldn't remove {path:?}");
     }
-    let accounts_db = AccountsDb::new_with_config_for_benches(
+    let accounts_db = AccountsDb::new_with_config(
```

Comment:
Not directly related to your PR, but I think this should be using`ClusterType::Development` instead ?

Reply:
Works for me